src/lib/CLValue/Bool.ts

```diff
@@ -12,16 +12,8 @@ import {
 import { BOOL_ID, CLTypeTag } from './constants';
 
 export class CLBoolType extends CLType {
-  linksTo = CLBool;
+  linksTo = BOOL_ID;
```

Comment:
Using _ID suffix is a bit cionfusing, it seems to indicate some kind of numerical identifier while actually it's a type. I know naming is hard but this does not seem an ID but a Type.

Reply:
I would suggest to grep and replace _ID by _TYPE but this could be a matter of taste.

Reply:
done :) thanks!